### DISCUSSION TITLE
etcd - wal: max entry size limit exceeded

### DISCUSSION THREAD

**Discussion by gabriel-montes-imprivata** [etcd-io/etcd#18648] [Q&A]
Hey,

Quick question: I was reviewing this issue (https://github.com/etcd-io/etcd/issues/18647) and wanted to see if anyone has experience upgrading etcd from one minor version to another.

We're planning to upgrade from 3.5.6 to 3.5.7—are there any key considerations or checks we should be aware of before proceeding?

Thanks!

**Comment by jmhbnz** [comment]
Hi @gabriel-montes-imprivata - Thanks for your question, is your cluster currently healthy?  Upgrading from `v3.5.6` --> `v.3.5.7` is only a patch version change so should be low risk.

For an example upgrade procedure you could refer to https://etcd.io/docs/v3.5/upgrades/upgrade_3_5/#upgrade-procedure.

Note:
1. The linked page is for a minor version bump i.e. `3.4.x` --> `3.5.x` which is a much more significant change than a simple patch version bump. 
2. You will likely need to adapt the instructions for your particular environment depending on how you deploy and manage your etcd cluster.

**Reply by gabriel-montes-imprivata** [reply]
We cannot deem our cluster healthy due to the issue reported here: https://github.com/etcd-io/etcd/issues/18647. Would this affect the upgrade process, or should we resolve the issue before proceeding with the upgrade?

Pura vida!

**Reply by ahrtr** [reply]
> We cannot deem our cluster healthy due to the issue reported here: #18647. Would this affect the upgrade process, or should we resolve the issue before proceeding with the upgrade?
> 
> Pura vida!

If you haven't run into https://github.com/etcd-io/etcd/issues/18647, then it should be OK to upgrade the version. Also it's recommended to bump the latest version, 3.5.17 for now.
